src/twist_mux_diagnostics.cpp

```diff
@@ -84,10 +89,7 @@ void TwistMuxDiagnostics::diagnostics(diagnostic_updater::DiagnosticStatusWrappe
               static_cast<int>(lock_h.getPriority()));
   }
 
-  stat.add("current priority", static_cast<int>(status_.priority));
-
-  stat.add("loop time in [sec]", status_.main_loop_time);
-  stat.add("data age in [sec]", status_.reading_age);
+  stat.add("current lock priority", static_cast<int>(status_.priority));
```

Comment:
now only `current lock priority` is displayed (however, this works)
why is the Key-Value pair for `current_priority` removed? Is this information not retrievable? (this would be the information I'm looking for)